packages/web-react/src/components/Footer/README.md

```diff
@@ -0,0 +1,268 @@
+# Footer
+
+The Footer is a highly variable composition. It comes in several ready-to-use design variants, but can be customized to
+achieve your specific design goals.
+
+## Composition
+
+The Footer supports several ready-to-use building blocks:
+
+- [Navigation Links](#navigation-links)
+- [Product Logo](#product-logo)
+- [Social Media Links](#social-media-links)
+- [Language Switch](#language-switch)
+- [Secondary Links](#secondary-links)
+
+This is how all supported building blocks of the Footer build up the complete composition:
+
+```jsx
+<Footer UNSAFE_className="bg-cover pt-1100 pb-1000">
+  <Container>
+    {/* Grid with navigation links */}
+    {/* Grid with product logo, social media links and language switch */}
+    {/* Flex with secondary links */}
+  </Container>
+</Footer>
+```
+
+👉 Good to know:
+
+- To achieve the desired design, you can use the provided building blocks in any order you need.
+- All building blocks are optional, so you can use only the ones you need.
+- You are not limited just to the provided building blocks. You can add your own content or building blocks (such as a
+  newsletter subscription form) as needed.
+- Use [Grid][grid] and [Flex][flex] components to create the desired layout.
+- Use [Stack][stack] or [Divider][divider] components to organize individual sections.
+- Use spacing utility classes like `mb-*` or `pt-*` to achieve desired spacings between components.
+
+## Navigation Links
+
+Navigation links are structured in sections with a headline and a [Stack][stack] of links.
+
+👉 Please note how the `<nav>` element is paired with an `<h3>` element (using the `aria-labelledby` attribute) to
+provide a semantic connection between the headline and the navigation component. Just make sure the `id` attribute of
+the `<h3>` element matches the value of the `aria-labelledby` attribute of the `<nav>` element and all `id`s are unique.
+
+```jsx
+<nav aria-labelledby="footer-navigation-section">
+  <h3 class="typography-body-large-text-bold mb-600" id="footer-navigation-section">
```

Comment:
We have a React component for this.

Reply:
I was not able to make it work with our `Heading` component. There is a `typography-body-large-text-bold` in web demo I followed, but our Heading component makes it `typography-heading-large-text-bold`. Even when I use the xsmall size, it is different from the web demo.

This might be a question for @adamkudrna about whether we should change it to `typography-heading-xsmall-text `in React and Web?

Reply:
After consulting with others, this will be done as `<Text elementType="h3" …>…</Text>`

Reply:
Discussed personally.